terms.html

```diff
@@ -55,7 +55,7 @@
   <dt><dfn data-lt="entities|entity's">entity</dfn></dt>
   <dd>
 A thing with distinct and independent existence, such as a person,
-organization, concept, or device. An entity can perform one or more roles in
+organization, or device that performs one or more roles in
```

Comment:
This change blows up a number of sentences in the specification. For example:

A <a>subject</a> is an <a>entity</a> about which <a>claims</a> can be made.

So, the change of the definition creates a cascading set of changes to the specification that are not going to be able to reconcile. :(

Reply:
Okay, maybe I was freaking out prematurely. I'd rather not make this change, but it looks like we have consensus in the issue among multiple people that regularly disagree about terminology, so that's something I don't want to get in the way of. :)

Merging, I'll make other non-normative changes to align other spec text to this new definition.

Reply:
Yeah, we used to say "An entity may perform"... and it was changed to "can" because people were concerned about the normative-ness of the statement, which subtly changed what the definition was saying. Anyway, merging. Editorial, I'll do the re-analysis of how the definition is used in the entire spec and make sure there are no normative changes as a result.